crates/stackable-versioned-macros/src/attrs/variant.rs

```diff
@@ -31,6 +31,12 @@ pub(crate) struct VariantAttributes {
     // shared item attributes because for struct fields, the type is
     // `Option<Ident>`, while for enum variants, the type is `Ident`.
     pub(crate) ident: Ident,
+
+    /// The original attributes for the field.
+    // This must be named `attrs` for darling to populate it accordingly, and
+    // cannot live in common because Vec<Attribute> is not implemented for
+    // FromMeta.
```

Comment:
suggestion: Same as above, flip the comments.

```suggestion
    // This must be named `attrs` for darling to populate it accordingly, and
    // cannot live in common because Vec<Attribute> is not implemented for
    // FromMeta.
    /// The original attributes for the field.
```